Source/items.cpp

```diff
@@ -710,8 +710,15 @@ int CalculateToHitBonus(int level)
 	}
 }
 
-int SaveItemPower(Item &item, const ItemPower &power)
+int SaveItemPower(Item &item, ItemPower &power)
 {
+	if (!gbIsHellfire) {
+		if (power.type == IPL_TARGAC) {
+			power.param1 = 1 << power.param1;

```

Comment:
I know these bitshifts were here before, but looking at piercing, puncturing, and bashing, this results in 2-8, 4-16, and 6-24, which doesn't match the original values.
![image](https://user-images.githubusercontent.com/6080734/153688657-9bc8e72e-3b48-41aa-b2e3-080def799df4.png)

Reply:
quite the opposite
3 << 1 = 6
3 << 2 = 12
3 << 3 = 24

Reply:
oops sorry about that, saw the parameters backwards ( `param1 << 1' etc ) by accident.